.github/workflows/build_image_ghcr.yml

```diff
@@ -0,0 +1,75 @@
+name: Build ghcr.io image
+
+on:
+  push:
+    branches: [ 'main', '*docker*', '*dev*' ]
+    tags:
+  workflow_dispatch:
+    inputs:
+      commit_sha:
+        description: 'Commit SHA to build'
+        required: true
+
+jobs:
+  build-amd64-image-to-ghcr:
+    runs-on: ubuntu-latest
```

Comment:
It's recommended to use 20.04 or 22.0 instead of lastet, because latest at a different time means a different version.

Reply:
updated in db75d099bf37b88ef7f443b6c697f3d02db22b83